Makefile

```diff
@@ -20,9 +20,9 @@ test: build
 host-build:
 	./checks "./*.go ./libpkt/ ./ofctrl/ ./ovsdbDriver/ ./ovsSwitch/ ./pqueue/ ./rpcHub/"
 	go get github.com/tools/godep
-	godep go install ./ ./ofctrl
+	go install ./ ./ofctrl
 
 host-test:
-	sudo -E PATH=$(PATH) /opt/gopath/bin/godep go test -v ./
-	sudo -E PATH=$(PATH) /opt/gopath/bin/godep go test -v ./ofctrl
-	sudo -E PATH=$(PATH) /opt/gopath/bin/godep go test -v ./pqueue
+	PATH=${PATH} sudo -E /usr/local/go/bin/go test -v ./
```

Comment:
@dseevr: This makes it easy for us to migrate away from godep. Even though we're not upgrading to Go 1.7, we're on Go 1.6 and we can get rid of godep in some places. The new box has Go in the same place, but it's easier to not rely on the box's gopath.